### DISCUSSION TITLE
求助Pake失败

### DISCUSSION THREAD

**Discussion by 17625261804** [tw93/Pake#514] [Q&A]
![image](https://github.com/tw93/Pake/assets/70893636/c0971481-8fd5-41ba-b3a5-10f5fc368a9c)

**Comment by tw93** [comment]
看起来是网络问题，需要试试github 代理

**Comment by 17625261804** [comment]
这是来自QQ邮箱的假期自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
